Template/Pages/Discipleship Training School.html

```diff
@@ -4,5 +4,20 @@
 [*
 <p>...</p>
 */*
-<p>...</p>
+<p> The Greek work SPITHA translates to SPARK, (which means; to arouse, kindle, trigger, instigate, excite, provoke, foment), or as some dictionaries would say: a feeling or quality that causes excitement or something that sets off a sudden force. The thought behind it was Isaiah 61:4 - 'They will rebuild the ancient ruins and restore the places long devastated; they will renew the ruined cities that have been devastated for generations.' This school will set off the spark and kindle the fire that will awaken the desire of the youth to rise up and stand with Jesus to rebuild the ancient ruins and to see the city of Thessaloniki, Greece restored to her true identity in Christ, as a nation and as a people. 'And now the word of the Lord is ringing out from you to people everywhere, even beyond Macedonia and Achaia, for wherever we go we find people telling us about your faith in God.' - 1 Thessalonians 1:8 </p>
+
+<p><b> Lecture Phase: </b></p>
+<p> The lecture phase is three months training of encountering Jesus through teaching, worship, prayer, service, practical discipleship and personal application of truth. Each week we'll focus on a different topic that is relevant to living out our faith in this generation and that will equip us in our pursuit of God. Come ready to be inspired in your pursuit of God and equipped to be a part of the greatest move of God the world has ever seen. </p>
+<p> Spitha DTS is for those who want to discover the fire that burns in the heart of God for His people and nations. Through a consecrated life, genuine intimacy and radical obedience to His Call you can change History! </p>
+
+<p><b> Outreach Phase: </b></p>
+<p> On outreach, you will encounter God's power working through you to transform the lives of those around you. You will have the opportunity to participate in prayer, personal witnessing, street evangelism, caring for the poor and many other initiatives that are relevant to the country you go to. The training you have received during the lecture phase will be relevant as you reach out and see people's lives transformed. You will see God working not only in the classroom but also in the nations. </p>
+
+<p><b> Dates and Cost: </b></p>
+<p> Start Date: 19th September, 2019 </p>
```

Comment:
This would be standard British/European style: (Yes, I know we new‐worlders would write “September 19, 2019” instead.)
```suggestion
<p> Start Date: 19 September 2019 </p>
```